sui/src/rest_gateway/requests.rs

```diff
@@ -63,13 +64,11 @@ pub struct CallRequest {
     /// Optional; The argument types to be parsed
     pub type_arguments: Option<Vec<String>>,
     /// Required; Byte representation of the arguments, Base64 encoded
-    pub pure_arguments: Vec<String>,
+    pub arguments: Vec<SuiJsonValue>,
     /// Required; Hex code as string representing the gas object id
     pub gas_object_id: String,
     /// Required; Gas budget required as a cap for gas usage
     pub gas_budget: u64,
-    /// Required; Object arguments
-    pub object_arguments: Vec<String>,
     /// Required; Share object arguments
     pub shared_object_arguments: Vec<String>,
```

Comment:
We should make this optional or maybe remove this as I would expect that it can also be inferred? @oxade to confirm